demo/meeting/src/meetingConfig.ts

```diff
@@ -20,12 +20,17 @@ const postLogConfig = {
   batchSize: 85,
   intervalMs: 2000,
   url: `${BASE_URL}logs`,
-  logLevel: SDK_LOG_LEVELS.info
+  logLevel: SDK_LOG_LEVELS.info,
+};
+
+const videoDownlinkBandwidthPolicyWrapper = (logger: Logger): VideoPriorityBasedPolicy => {
```

Comment:
To create a downlink policy, we need to pass in a `Logger`. In React SDK we create the `Logger` in `MeetingManager` `initializeMeetingSession` function. To allow user customize the downlink policy we need to wrap it.

Reply:
Can we not pass the custom `VideoPriorityBasedPolicy` directly?

Also, if this `Logger` is created separately here, we would need a way to use this same while creating the meeting session right? Otherwise, there would be separate loggers one created in the default meeting session creation and this one.

Reply:
I post the solution below. You are right, if we want to pass the custom `VideoPriorityBasedPolicy` directly, we need to initialize the logger in the `MeetingConfig.ts`. Then we need to handle the conflict of two loggers.